### ISSUE TITLE
Error after joining voice.

### ISSUE THREAD

**Issue by macdja38** [hydrabolt/discord.js#350]
this happened just after binding to a channel
```
pvpcraft-14 TypeError: Cannot read property 'id' of null
pvpcraft-14     at WebSocket.websocket.onmessage (/Bots/pvpcraft/node_modules/discord.js/lib/Client/InternalClient.js:2255:48)
pvpcraft-14     at WebSocket.onMessage (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:442:14)
pvpcraft-14     at emitTwo (events.js:111:20)
pvpcraft-14     at WebSocket.emit (events.js:191:7)
pvpcraft-14     at Receiver.ontext (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:841:10)
pvpcraft-14     at /Bots/pvpcraft/node_modules/ws/lib/Receiver.js:536:18
pvpcraft-14     at Receiver.applyExtensions (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:371:5)
pvpcraft-14     at /Bots/pvpcraft/node_modules/ws/lib/Receiver.js:508:14
pvpcraft-14     at Receiver.flush (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:347:3)
pvpcraft-14     at Receiver.opcodes.1.finish (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:541:12)
pvpcraft-14     at Receiver.expectHandler (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:499:31)
pvpcraft-14     at Receiver.add (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:103:24)
pvpcraft-14     at TLSSocket.realHandler (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:825:20)
pvpcraft-14     at emitOne (events.js:96:13)
pvpcraft-14     at TLSSocket.emit (events.js:188:7)
pvpcraft-14     at readableAddChunk (_stream_readable.js:172:18)
pvpcraft-14     at TLSSocket.Readable.push (_stream_readable.js:130:10)
pvpcraft-14     at TLSWrap.onread (net.js:542:20)
```

**Comment by macdja38** [comment]
may be an internet problem? this also happened
```
pvpcraft-14 Error: Not running
pvpcraft-14     at Socket._healthCheck (dgram.js:512:11)
pvpcraft-14     at Socket.close (dgram.js:406:8)
pvpcraft-14     at VoiceConnection.destroy (/Bots/pvpcraft/node_modules/discord.js/lib/Voice/VoiceConnection.js:98:12)
pvpcraft-14     at player.destroy (/Bots/pvpcraft/lib/player.js:38:29)
pvpcraft-14     at music.onDisconnect (/Bots/pvpcraft/modules/music.js:36:39)
pvpcraft-14     at reload (/Bots/pvpcraft/main.js:195:31)
pvpcraft-14     at Client.client.on (/Bots/pvpcraft/main.js:237:5)
pvpcraft-14     at emitNone (events.js:86:13)
pvpcraft-14     at Client.emit (events.js:185:7)
pvpcraft-14     at InternalClient.checkReady (/Bots/pvpcraft/node_modules/discord.js/lib/Client/InternalClient.js:450:17)
pvpcraft-14     at WebSocket.websocket.onmessage (/Bots/pvpcraft/node_modules/discord.js/lib/Client/InternalClient.js:2300:14)
pvpcraft-14     at WebSocket.onMessage (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:442:14)
pvpcraft-14     at emitTwo (events.js:106:13)
pvpcraft-14     at WebSocket.emit (events.js:191:7)
pvpcraft-14     at Receiver.ontext (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:841:10)
pvpcraft-14     at /Bots/pvpcraft/node_modules/ws/lib/Receiver.js:536:18
pvpcraft-14     at Receiver.applyExtensions (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:371:5)
pvpcraft-14     at /Bots/pvpcraft/node_modules/ws/lib/Receiver.js:508:14
pvpcraft-14     at Receiver.flush (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:347:3)
pvpcraft-14     at Receiver.opcodes.1.finish (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:541:12)
pvpcraft-14     at Receiver.expectHandler (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:499:31)
pvpcraft-14     at Receiver.add (/Bots/pvpcraft/node_modules/ws/lib/Receiver.js:103:24)
pvpcraft-14     at TLSSocket.realHandler (/Bots/pvpcraft/node_modules/ws/lib/WebSocket.js:825:20)
pvpcraft-14     at emitOne (events.js:96:13)
pvpcraft-14     at TLSSocket.emit (events.js:188:7)
pvpcraft-14     at readableAddChunk (_stream_readable.js:172:18)
pvpcraft-14     at TLSSocket.Readable.push (_stream_readable.js:130:10)
pvpcraft-14     at TLSWrap.onread (net.js:542:20)
```

**Comment by macdja38** [comment]
ok, so I believe that the first problem is caused by having 2 entities logged into the same account. one joining a voice channel can cause the other to crash with the first error.

**Comment by abalabahaha** [comment]
Should be fixed, I think in e180880
